src/main/kotlin/no/nav/syfo/db/NlDb.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.syfo.db
+
+import kotliquery.*
+import no.nav.syfo.getEnvVar
+import no.nav.syfo.getEnvironment
+import no.nav.syfo.narmestelederapi.NarmesteLeder
+import org.flywaydb.core.Flyway
+import java.time.LocalDate
+import java.time.ZonedDateTime
+
+val toNarmesteLeder: (Row) -> NarmesteLeder = { row ->
+    NarmesteLeder(
+            row.string("aktorid"),
+            row.string("orgnummer"),
+            row.string("nlaktorid"),
+            row.string("nltelefonnummer"),
+            row.string("nlepost"),
+            row.sqlDate("aktivfom").toLocalDate(),
+            row.boolean("agforskutterer")
+    )
+}
+
```

Comment:
Du kan skille dette i to biter:
Flyway og HikariCP biten kan gjøres i Bootstrap.kt og returnere en session

Denne klassen kan du beholde som narmestelederDAO som tar inn en session og beholde metodene for å hente ut data